JSDemos/Demos/NumberBox/Formatting/description.md

```diff
@@ -1 +1,15 @@
-This demo shows how to&nbsp;specify the currency, accounting, percent and other formats using the [format](/Documentation/ApiReference/UI_Components/dxNumberBox/Configuration/#format) property. It&nbsp;makes the NumberBox not only format the display value, but also control the user input.
\ No newline at end of file
+Use the [format](/Documentation/ApiReference/UI_Components/dxNumberBox/Configuration/#format) property to specify the currency, accounting, percent, and other formats of the NumberBox value.
+
+The format property can accept the following value types:
+
+- Predefined format    
+See the [type](/Documentation/ApiReference/Common/Object_Structures/format/#type) property description for a list of accepted values.
+
+- Custom format string    
+Use a LDML pattern to specify a custom format. Refer to the following article for more information: [Custom Format String](/Documentation/Guide/Common/Value_Formatting/#Format_UI_Component_Values/Custom_Format_String).
```

Comment:
```suggestion
Use an LDML pattern to specify a custom format. Refer to the following article for more information: [Custom Format String](/Documentation/Guide/Common/Value_Formatting/#Format_UI_Component_Values/Custom_Format_String).
```

Changed 'a' to 'an' because when you pronounce this abbreviation, the 'L' is 'ELL'